hooks/pre-commit

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright © 2019 Martin Ueding <dev@martin-ueding.de>
+
+set -e
+set -u
+
+red="$(tput setaf 1)"
+reset="$(tput sgr0)"
+
+diff_old=$(mktemp)
+diff_new=$(mktemp)
+
+cleanup() {
+  rm -f "$diff_old"
+  rm -f "$diff_new"
+}
+
+trap cleanup EXIT
+
+git diff > "$diff_old"
+Rscript -e 'devtools::document();'
+git diff > "$diff_new"
+
+if ! diff -u "$diff_old" "$diff_new" &> /dev/null; then
```

Comment:
I have added `-q` because the output is ignored anyway. But doesn't every Linux distribution have `/dev/null`? And the execution of this shell script needs a Bash, and I doubt that Johann has a Bash but not `/dev/null` in the Linux on Windows.

Reply:
I see, that's a typo then. You wrote ```diff -u``` above and the argument for ```-u``` is missing.

Reply:
I agree that you can assume the existence of /dev/null if a bash is present